deploy/deploy.py

```diff
@@ -102,3 +102,39 @@ def delete():
 
     # currently just text
     return jsonify({'text': response.stdout.decode('utf-8')})
+
+
+@app.route('/api/redeploy', methods=['POST'])
+@requires_auth
+def redploy():
+    request_data = request.get_json()
+    data = dict(username=request_data['github'])
+
+    # Delete the user
+    try:
+        response = commands.delete_user(args=data)
+    except subprocess.CalledProcessError as e:
+        app.logger.error('Error calling deploy.sh: %s', str(e.output))
+        return Response('Error calling deploy', 500)
+
+    # Delete the app
+    try:
+        data['chart'] = 'rstudio'
+        response = commands.delete_chart(args=data)
+    except subprocess.CalledProcessError as e:
+        app.logger.error('Error calling deploy.sh: %s', str(e.output))
+        return Response('Error calling deploy', 500)
+
+    # Deploy the app again
+    data = dict(
+        fullname='redeploy',
```

Comment:
Might be worth a comment to explain that fullname and email are not used for anything...